authors/apps/articles/serializers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+
+from .models import Article
+
+
+class ArticleSerializer(serializers.ModelSerializer):
+    """Serializer class for the story's model."""
+
+    class Meta:
+        """class for returning our field."""
+
+        model = Article
+        fields = '__all__'
+        read_only_fields = ('created_at','updated_at','favorited','favorite_count','slug')
```

Comment:
Endeavour to lint your code before you push to avoid no newline errors, and crowded code.